scripts/setup.sh

```diff
@@ -43,6 +43,11 @@ function maybeInstallDependencies {
       shift
       ;;
 
+      --suite=*)
```

Comment:
I'm sorry, I still don't understand. To be more specific, my question is: what is the difference between 'suite' and 'sharding'? You say that 'sharding' never reaches setup.sh but 'suite' does; can you explain why?